quickwit/quickwit-config/src/node_config/serialize.rs

```diff
@@ -59,6 +59,10 @@ fn default_node_id() -> ConfigValue<String, QW_NODE_ID> {
     ConfigValue::with_default(node_id)
 }
 
+fn default_enable_elastic_header() -> ConfigValue<bool, QW_ENABLE_ELSTIC_HEADER> {
```

Comment:
Given the usage of the elastic header, I would prefer to not add a new env to manage.